projects/adrv9009_zu11eg_som/carrier_bd.tcl

```diff
@@ -1,2 +1,14 @@
 
 add_files -fileset constrs_1 -norecurse ./carrier_constr.xdc
+
+create_bd_port -dir I axi_fan_tacho_i
+create_bd_port -dir O axi_fan_pwm_o
+
+ad_ip_instance axi_fan_control axi_fan_control_0
+ad_ip_parameter axi_fan_control_0 CONFIG.ID 1
+
+ad_connect axi_fan_tacho_i axi_fan_control_0/tacho
+ad_connect axi_fan_pwm_o axi_fan_control_0/pwm
+
+ad_cpu_interrupt ps-14 mb-14 axi_fan_control_0/irq
+ad_cpu_interconnect 0x80000000 axi_fan_control_0
```

Comment:
Can you use an address starting with 0x4 for the IP, this will make the addressing similar for the whole system. Using 0x40000000 will actually result in the system address 0x80000000  as 0x40000000 is added for ZynqMP systems.

Reply:
done